projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.html

```diff
@@ -0,0 +1,12 @@
+<!-- TODO: Verify current user to display left aligned messages -->
+<div
+  *ngIf="event"
```

Comment:
This is a shared component that uses properties from a Customer Ticketing Event -> too coupled.
Let's change "event" by  "message"

Reply:
got it